### ISSUE TITLE
Docker container is huge

### ISSUE THREAD

**Issue by aileenzeng** [jonfroehlich/makeabilitylabwebsite#595]
When I build my container from scratch, the image for the website is ~35 GB, which takes 5+ minutes for me to build. (In the past, our Docker container was just over 1 GB).

**Comment by jonfroehlich** [comment]
Oh no! Are there Docker tools that allow us to investigate content within a container?

**Comment by aileenzeng** [comment]
`docker exec -it <container name> bash` lets us open up an interactive bash terminal inside a running container. From there, we can go look at what sorts of files are in our docker container. I can't do it right now since I'm still building the image, but I can take a peek later!

**Comment by jonfroehlich** [comment]
Oops! I wonder how that happened? Maybe the talks you just imported?

**Comment by aileenzeng** [comment]
Yeah - just saw that. All the talks from `importtalks` in #592 are getting sent to our build context (but I don't think it has to).

**Comment by jonfroehlich** [comment]
Agree. And this is only for your dev environment, right?

**Comment by aileenzeng** [comment]
Yup -- looks like the size mostly varies w/ how much stuff is in my local media directory.

**Comment by aileenzeng** [comment]
I think I found a solution, but I can't test right now because I can't run the website anymore because I can't connect to the database -- currently writing up an issue.

**Comment by jonfroehlich** [comment]
@aileenzeng, I think I'll assign this one to you since you've been our resident docker magician/expert.
